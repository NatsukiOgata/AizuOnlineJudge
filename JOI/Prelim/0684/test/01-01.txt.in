99 100
65 14 45 82 27 27 14 87 23 14 82 82 27 87 87 57 14 3 3 14 3 82 61 45 3 45 23 14 65 87 27 14 23 14 14 57 3 45 57 45 65 65 57 82 3 14 45 82 82 57 14 82 57 61 57 14 27 27 23 23 14 27 45 27 3 27 57 87 82 27 65 23 65 65 45 57 45 23 45 87 65 87 45 87 45 61 3 45 45 45 14 14 61 61 27 57 45 3 23
65 14 82 23 57 45 87 45 57 45 45 3 82 87 57 14 61 23 87 14 57 45 14 82 45 3 23 61 82 61 27 82 45 65 3 14 57 23 14 57 27 3 87 82 45 45 65 14 45 45 82 27 27 45 61 14 87 45 65 61 87 82 82 82 82 27 27 14 57 45 57 61 82 23 82 3 27 65 57 87 87 61 23 82 3 87 14 61 87 27 82 14 61 45 87 27 45 87 87 23
