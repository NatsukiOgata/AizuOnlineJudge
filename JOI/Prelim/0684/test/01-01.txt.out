3
14
23
27
45
57
61
65
82
87
